components/sidebar/index.jsx

```diff
@@ -53,8 +53,8 @@ const Sidebar = React.memo(({ className }) => {
             <Link href={route.path}>
               <a className={sidebarClassNames.route} href="/">
                 <Icon
-                  className={sidebarClassNames.sidebarIcon}
-                  iconType={route.icon}
+                  alt={`${route.title} icon`}
+                  src={`/design/icons.svg#${route.icon}`}
```

Comment:
Maybe create a utility function for this or another component that does this automatically. Also, I am considering a CDN solution

Reply:
Definitely I support it. CDN would solve this issue https://github.com/oacore/dashboard/pull/19#discussion_r356657904 too.